src/arkit.cs

```diff
@@ -67,7 +67,9 @@ public enum ARErrorCode : long {
 		InvalidReferenceObject = 301,
 		InvalidWorldMap = 302,
 		InvalidConfiguration = 303,
+		[Deprecated (PlatformName.iOS, 13, 2, message : "Please use the 'InvalidCollaborationData' value instead.")]
 		CollaborationDataUnavailable = 304,
+		InvalidCollaborationData = 304,
```

Comment:
missing availability attribute.

Reply:
But its an error enum. https://github.com/xamarin/xamarin-macios/pull/7322#discussion_r339817547 Does it make sense to add it then?